nbgrader/exchange/list.py

```diff
@@ -79,20 +79,36 @@ def parse_assignments(self):
                 info['status'] = 'removed'
 
             info['notebooks'] = []
+            hasFeedback = False
             for notebook in sorted(glob.glob(os.path.join(info['path'], '*.ipynb'))):
-                info['notebooks'].append({
+                nbInfo = {
                     'notebook_id': os.path.splitext(os.path.split(notebook)[1])[0],
                     'path': os.path.abspath(notebook)
-                })
-
+                }
+                nb_hash = notebook_hash(notebook)
+                _, notebookFilename = os.path.split(notebook)
+                notebookName, _ = os.path.splitext(notebookFilename)
+                feedbackpath = os.path.join(self.root, info['course_id'], 'feedback', '{0}.html'.format(nb_hash))  
+                localFeedbackpath = os.path.join(self.coursedir.root, info['course_id'], 'feedback', '{0}.html'.format(notebookName))  
```

Comment:
@danielmaitre I believe that the path assumes that the files are `course_id/feedback` but that should be either:

1. `course_id/assignment_id/feedback` (if organized by course), or
2. `assignment_id/feedback` (if not organized by course)

Reply:
The current `fetch.py` code in the exchange does:

```python
        if self.path_includes_course:
            root = os.path.join(self.course_id, self.coursedir.assignment_id)
        else:
            root = self.coursedir.assignment_id
        self.dest_path = os.path.abspath(os.path.join('.', root))
```

Reply:
I'll fix the merge confilcts

Reply:
Yes the local path was not set properly, I wrote this bit before we decided what directory structure we wanted, and forgot to adapt it. I've now changed it and added the timestamp as well so now the UI shows "view" instead of available when the file is present.

Reply:
fixed a few more missing @notwindows

Reply:
Looks good! I'm turning the "view" into a link, and the "available" to "ready to fetch". Will make a PR within the next hour or so.

Reply:
PR adds link to "view", and changes text to "ready to fetch": https://github.com/danielmaitre/nbgrader/pull/9